routes/sessions.js

```diff
@@ -0,0 +1,35 @@
+var express = require('express');
+var router = express.Router();
+var User = require('./../models').User;
+
+const bcrypt = require('bcrypt');
+const saltRounds = 10;
+const uuidv4 = require('uuid/v4');
+
+router.post('/', function(req, res, next) {
+  if (req.body.email && req.body.password) {
+    User.findOne({
+      where: {
+        email: req.body.email
+      }
+    })
+    .then(user => {
+      if (bcrypt.compareSync(req.body.password, user.password)) {
+        res.setHeader("Content-type", "application/json");
+        res.status(200).send(JSON.stringify({apiKey: user.apiKey}));
+      } else {
+        res.setHeader("Content-type", "application/json");
+        res.status(401).send(JSON.stringify("Incorrect Password"));
```

Comment:
On lines 22, 27 & 31 - in the `.send(JSON.stringify("...."));` method - you pass in a value into the argument (i.e. "Incorrect Password"), but the key is missing (i.e. "error").